optimum/exporters/openvino/convert.py

```diff
@@ -412,11 +414,24 @@ def ts_patched_forward(*args, **kwargs):
 
         if stateful:
             patch_stateful(model.config, ov_model)
+        if ov_config.quantization_config:
```

Comment:
there is another PR where we probably solve this problem: https://github.com/huggingface/optimum-intel/pull/721/files

Reply:
Yes, seems like the changes are independent.

@eaidova Just out of curiosity, do you have numbers for how much memory can be saved with this approach?

Reply:
thanks, @eaidova. @nikita-savelyevv, can you please adopt changes from this PR?

Reply:
@AlexKoff88 I don't think this applies to my case because I don't have access to the PyTorch or OpenVINO model objects after the `main_export` call. So I can't delete them.

Or do you mean to copy changes from this PR to my PR? If so, in my opinion these should be added separately.